.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,10 @@
 
 ## The Issue
 
+- #<issue number>
```

Comment:
Maybe update to `- Fixes #<issue number>` here?

Reply:
I don't know, it's not always the situation when the PR fixes something.
From what I see for the PRs I make, I usually add a link here to the issue or another PR or Discord.

But the title must have `fixes` in it if possible, which I find very useful because I use "blame" feature in GitLens with VSCode and in GitToolbox with JetBrains, and when I hover over any line, I see the commit author and the message, and I instantly know which issue/PR is associated with the change I'm looking at.